src/background/config_rc.ts

```diff
@@ -59,17 +59,21 @@ export async function runRc(rc: string) {
 }
 
 export function rcFileToExCmds(rcText: string): string[] {
-    // string-join lines that end with /
-    const joined = rcText.replace(/\\\n/g, "")
+   // Remove empty and comment lines
+   const rm= Array.from(rcText).filter(
+    x =>
+        /\S/.test(x) &&
+        !x.trim().startsWith('"') &&
+        !x.trim().startsWith("#"),
+   )
+   // Converting from an array to strings  
+   const str = rm.join(" ");
```

Comment:
Here I tried to convert the array back to string so as I could perform the string operations in the later stages. @bovine3dom

Reply:
I don't think that's the right approach. Your command would add spaces where there were none before.